app/src/main/java/at/aau/serg/models/LobbyJoin.kt

```diff
@@ -0,0 +1,3 @@
+package at.aau.serg.models
+
+data class LobbyJoin(val lobbyID: String)
```

Comment:
Not sure if we need models for a single field

Reply:
not needed but not a problem i think we should use models for all parsed bodies